content/docs/using-pulumi/inspecting-infrastructure/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title_tag: "Inspecting Infrastructure | Tutorials"
+meta_desc: Learn how to inspect your Pulumi resources in this tutorial.
+title: "Inspecting Infrastructure"
+h1: "Inspecting Infrastructure"
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+  usingpulumi:
+    identifier: inspecting-infrastructure
+    weight: 1
+tags:
+    - resources
+    - infrastructure
+    - fundamentals
+    - tutorial
+---
+
+Pulumi configurations and state data include highly structured information about the resources they manage, such as dependency information, [outputs](/docs/concepts/inputs-outputs/#outputs), and more.
+
+The Pulumi CLI includes some commands for inspecting this data. You can use these to integrate other tools with Pulumi's infrastructure data, or just to gain a deeper or more holistic understanding of your infrastructure.
+
+In this tutorial, we'll demonstrate the various ways you can use the Pulumi CLI to inspect your infrastructure.
+
+## Prerequisites
+
+This tutorial assumes that you are familiar with the basics of the Pulumi workflow. If you are new to Pulumi, complete the [Get Started series](/docs/get-started/) first.
+
+Additionally, you will need the following tools to complete this tutorial:
+
+- A [Pulumi account](https://app.pulumi.com)
+  - [Optional] Create an [access token](/docs/pulumi-cloud/access-management/access-tokens/)
+- The [Pulumi CLI](https://www.pulumi.com/docs/install/)
+- An [Amazon Web Services](https://aws.amazon.com/) account
+- The [AWS CLI](https://aws.amazon.com/cli/)
+- Your desired [language runtime installed](/docs/clouds/aws/get-started/begin/#install-language-runtime)
+
+Let's get started!
+
+## Create a New Project
+
+To start, login to the [Pulumi CLI](/docs/cli/commands/pulumi_login/) and ensure it is [configured to use your AWS account](/docs/clouds/aws/get-started/begin/#configure-pulumi-to-access-your-aws-account). Next, [create a new project](/docs/clouds/aws/get-started/create-project/) and replace the default program code with the following:
```

Comment:
Perhaps we explain what this code does ahead of time? and/or also reference that you can run through all these steps on any existing project/stack you already have created?

Reply:
updated